crates/vm/levm/src/gas_cost.rs

```diff
@@ -748,6 +760,103 @@ pub fn fake_exponential(factor: u64, numerator: u64, denominator: u64) -> Result
     ))
 }
 
+pub fn sha2_256(data_size: u64) -> Result<U256, OutOfGasError> {
+    precompile(data_size, SHA2_256_STATIC_COST, SHA2_256_DYNAMIC_BASE)
+}
+
+pub fn ripemd_160(data_size: u64) -> Result<U256, OutOfGasError> {
+    precompile(data_size, RIPEMD_160_STATIC_COST, RIPEMD_160_DYNAMIC_BASE)
+}
+
+pub fn identity(data_size: u64) -> Result<U256, OutOfGasError> {
+    precompile(data_size, IDENTITY_STATIC_COST, IDENTITY_DYNAMIC_BASE)
+}
+
+pub const MODEXP_STATIC_COST: u64 = 0;
+pub const MODEXP_DYNAMIC_BASE: u64 = 200;
+pub const MODEXP_DYNAMIC_QUOTIENT: u64 = 3;
```

Comment:
We can move this to the top of the file

Reply:
Done in [Move constants to top in gas cost](https://github.com/lambdaclass/ethrex/pull/1454/commits/50167886ec065702616fea14b4f38a7b34b5cf40)